TrafficCapture/trafficReplayer/README.md

```diff
@@ -140,8 +148,9 @@ To run only one transformer without any configuration, the `--transformer-config
 be set to the name of the transformer (e.g. 'JsonTransformerForOpenSearch23PlusTargetTransformerProvider', 
 without quotes or any json surrounding it).
 
-The user can also specify a file to read the transformations from using the `--transformer-config-file`, but can't use
-both transformer options.
+The user can also specify a file to read the transformations from using the `--transformer-config-file`.  Users can
```

Comment:
I'm not w/ the CDK deployment today, which is why I put a Base64 option in there too.  Further out, it might make sense to pull it via a parameter store and write it as part of an init task.  I'm going to try not bother with changing things like container initialization until I'm done w/ the K8s investigation.

Reply:
Makes sense, thanks.